app/sls-tools.js

```diff
@@ -83,6 +83,16 @@ const runExecute = async (cmdArgs) => {
     await _handleCmdOperation(cmdArgs, executeCommandUsage, execute);
 }
 
+/**
+ * Run execute without env command entry point
+ *
+ * @param cmdArgs
+ * @returns {Promise<void>}
+ */
+const runExecuteNoEnv = async (cmdArgs) => {
+    await _handleCmdOperation(cmdArgs, executeCommandUsage, execute);
```

Comment:
we should use different ```execute``` implementation which will not process the configuration at all